pages/en/lb4/Getting-started.md

```diff
@@ -5,35 +5,202 @@ keywords: LoopBack 4.0, LoopBack 4
-- How to install the command line utility
-- Using the utility to scaffold the skeleton app
-- Running the app (node ., npm start)
+### Create `package.json`
+
+Regardless of whether you're going to code in JavaScript or TypeScript, you need to create  a `package.json` file.
+The easiest way to create a new one is to enter the following command:
+
+  ```
+  $ npm init
+  ```
+
+The command will prompt you for values that it will use to fill out the `pacakge.json` properties.  Press ENTER to accept the default for all of them, except for `description` enter `Hello World for LoopBack 4`.
+You can also enter values for entries like `git repository` and `keywords`,
+but they are not required.
+
+```
+package name: (lb4-hello-world)
+version: (1.0.0)
+description: Hello World for LoopBack 4
+entry point: (index.js)
+test command:
+git repository:
+keywords:
+author:
+license: (ISC)
+```
+
+The file will then contain the following:
+
+```js
+{
+  "name": "lb4-hello-world",
+  "version": "1.0.0",
+  "description": "Hello World for LoopBack 4",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC"
+}
+```
+
+**To continue, follow one of these procedures**:
+
+- [JavaScript project](#javascript-project)
+- [TypeScript project](#typescript-project)
+
+## Try a JavaScript project
+
+If you're coding in JavaScript, follow these steps:
+
+1. Create `index.js` file.
+1. Run the project.
+
+### Create `index.js`
+
+Create a file named `index.js`, and copy the following into it:
+
+```js
+const Application = require('@loopback/core').Application;
+
+const app = new Application();
+app.bind('message').to('Hello worldA');
+app.get('message').then(value => {
+  console.log(value);
+});
+```
+
+### Run the project
+
+To run your project, enter this command:
+
+```shell
+node index.js
+```
+
+You should see "Hello world!" written to the console.
+
+## Try a TypeScript project
+
+If you're coding in TypeScript, follow these steps:
+
+1. Create `tsconfig.json` file.
+2. Create `index.ts` file.
+1. Run the project.
+
+### Create `tsconfig.json` file
+
+A `tsconfig.json` file in a directory indicates that it contains a TypeScript project. The `tsconfig.json` file specifies the root files and the TypeScript compiler options. For more information on `tsconfig.json`, see [TypeScript documentation](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html).
+
+Create a new `tsconfig.json` file by entering the following command:
+
+```
+$ tsc --init
+```
+
+Now edit this file, because you need to change some of the default values.
+
+{% include important.html content= "You must set the `'compilerOptions.target'` property in  your project's `tsconfig.json` file based on the version of Node you are using.
+" %}
+
+**With Node 6.x**:
+
+```js
+{
+  "compilerOptions": {
+   ...
+    "target": "es6",
+    ...
+  }
+}
+```
+
+**With Node 8.x**:
```

Comment:
Nitpick: can we reverse the order and show instructions for Node.js 8.x first, 6.x second?

Reply:
Done.